logfire/_internal/ast_utils.py

```diff
@@ -68,21 +68,36 @@ def rewrite_function(self, node: ast.FunctionDef | ast.AsyncFunctionDef, qualnam
         # Replace the body of the function with:
         #     with <logfire_method_call_node>:
         #         <original body>
+        body = node.body.copy()
+        new_body: list[ast.stmt] = []
+        if (
+            body
+            and isinstance(body[0], ast.Expr)
```

Comment:
I suppose it is guaranteed that `0` must exist (even if it is just `pass` or `...`)?

Reply:
I'm pretty sure it is, but I started the condition with `body and` just in case.